cmd/deploy/deploy.go

```diff
@@ -417,7 +421,13 @@ func (dc *deployCommand) showEndpoints(ctx context.Context, opts *Options) error
 		sort.Slice(eps, func(i, j int) bool {
 			return len(eps[i]) < len(eps[j])
 		})
-		log.Information("Endpoints available:\n  - %s\n", strings.Join(eps, "\n  - "))
+		switch log.GetOutputFormat() {
+		case "json":
+			log.Println(fmt.Sprintf("Endpoints available:\n  - %s\n", strings.Join(eps, "\n  - ")))
```

Comment:
I think we should be able to simplify this and just use a single log.Printf without the Sprintf and Join call.